.husky/generate-docs

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
```

Comment:
I had to rename the file to `.husky/pre-commit`
but it's currently generating `docs/admin-api.md` but not staging the changes.

Reply:
> I had to rename the file to `.husky/pre-commit`

I did now, too. But I'm pretty sure it worked the first time. 🤷‍♀️

> but it's currently generating docs/admin-api.md but not staging the changes.
Yes, same problem. I'll investigate. 

Maybe it didn't work before and I just thought it did...